spec/features/company_view_spec.rb

```diff
@@ -69,68 +69,68 @@
     end
   end
 
-  #describe "#email_addresses" do
+  describe "#email_addresses" do
 
-    #before(:each) do
-      #company.email_addresses.create(address: "molsen@example.com")
-      #company.email_addresses.create(address: "belieber@example.com")
+    before(:each) do
+      company.email_addresses.create(address: "molsen@example.com")
+      company.email_addresses.create(address: "belieber@example.com")
 
-      #visit company_path(company)
-    #end
+      visit company_path(company)
+    end
 
-    #it "shows the email addresses" do
-      #company.email_addresses.each do |email|
-        #expect(page).to have_selector('li', text: "#{email.address}")
-      #end
-    #end
+    it "shows the email addresses" do
+      company.email_addresses.each do |email|
+        expect(page).to have_selector('li', text: "#{email.address}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.